src/modules/client.ts

```diff
@@ -95,14 +95,14 @@ export default class Client {
 
   subscribe(
     callback: (changedTypes: string[], response: object) => void
-  ): string {
+  ): number {
     // Create an identifier, add callback to subscriptions, return identifier
-    const id = uuid();
+    const id = this.subscriptionSize++;
     this.subscriptions[id] = callback;
     return id;
   }
 
-  unsubscribe(id: string) {
+  unsubscribe(id: number) {
```

Comment:
@kenwheeler Couldn't we also adopt the pattern of returning an unsubscribe callback from the `subscribe` method? I'd love the simplicity of applying that.